gulpfile.js

```diff
@@ -1,26 +1,46 @@
 var elixir = require('laravel-elixir');
```

Comment:
@vinicius73  I agree with you, approve the PR in this moment. And i will refactor, i will work in others improves :)